tests/components/lektrico/test_config_flow.py

```diff
@@ -0,0 +1,175 @@
+"""Tests for the Lektrico Charging Station config flow."""
+
+import dataclasses
+from ipaddress import ip_address
+
+from lektricowifi import DeviceConnectionError
+
+from homeassistant.components.lektrico.const import DOMAIN
+from homeassistant.config_entries import SOURCE_USER, SOURCE_ZEROCONF
+from homeassistant.const import (
+    ATTR_HW_VERSION,
+    ATTR_SERIAL_NUMBER,
+    CONF_HOST,
+    CONF_NAME,
+    CONF_TYPE,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+from .conftest import (
+    MOCKED_DEVICE_BOARD_REV,
+    MOCKED_DEVICE_IP_ADDRESS,
+    MOCKED_DEVICE_SERIAL_NUMBER,
+    MOCKED_DEVICE_TYPE,
+    MOCKED_DEVICE_ZEROCONF_DATA,
+)
+
+from tests.common import MockConfigEntry
+
+
+async def test_user_setup(
+    hass: HomeAssistant, mock_device_config, mock_setup_entry
+) -> None:
+    """Test manually setting up."""
+
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN,
+        context={"source": SOURCE_USER},
+    )
+
+    assert result["type"] is FlowResultType.FORM
+    assert result["step_id"] == SOURCE_USER
+    assert "flow_id" in result
+
+    result = await hass.config_entries.flow.async_configure(
+        result["flow_id"],
+        {
+            CONF_HOST: MOCKED_DEVICE_IP_ADDRESS,
+        },
+    )
+    await hass.async_block_till_done()
+
+    assert result.get("type") is FlowResultType.CREATE_ENTRY
+    assert result.get("title") == f"{MOCKED_DEVICE_TYPE}_{MOCKED_DEVICE_SERIAL_NUMBER}"
+    assert result.get("data") == {
+        CONF_HOST: MOCKED_DEVICE_IP_ADDRESS,
+        CONF_NAME: f"{MOCKED_DEVICE_TYPE}_{MOCKED_DEVICE_SERIAL_NUMBER}",
+        ATTR_SERIAL_NUMBER: MOCKED_DEVICE_SERIAL_NUMBER,
+        CONF_TYPE: MOCKED_DEVICE_TYPE,
+        ATTR_HW_VERSION: MOCKED_DEVICE_BOARD_REV,
+    }
+    assert "result" in result
+    assert len(mock_setup_entry.mock_calls) == 1
+
+    entries = hass.config_entries.async_entries(DOMAIN)
+    assert entries[0].unique_id == MOCKED_DEVICE_SERIAL_NUMBER
```

Comment:
You can just check the result variable here

Reply:
I don't understand what I should change or add.

Reply:
I made the change, thanks.